src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/WindowsLocalDebugger.xaml

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule
+	Name="WindowsLocalDebugger"
+	DisplayName="Windows Local Debugger"
+	PageTemplate="debugger"
+	Description="Windows Local Debugger options"
+    OverrideMode="Replace"
+	xmlns="http://schemas.microsoft.com/build/2009/properties">
+    <Rule.Categories>
+        <Category Name="General" DisplayName="General" Description="General" />
+    </Rule.Categories>
+    <Rule.DataSource>
+        <DataSource Persistence="UserFileWithXamlDefaults" Label="Configuration" />
+    </Rule.DataSource>
+
+    <StringProperty Name="LocalDebuggerCommand" >
+        <StringProperty.DataSource>
+            <DataSource Persistence="UserFileWithXamlDefaultsWithInterception"/>
+        </StringProperty.DataSource>
+    </StringProperty>
+    <StringProperty Name="LocalDebuggerCommandArguments" >
+        <StringProperty.DataSource>
+            <DataSource Persistence="UserFileWithXamlDefaultsWithInterception"/>
+        </StringProperty.DataSource>
+    </StringProperty>
+    <!--<StringProperty Name="LocalDebuggerDebuggerType">
```

Comment:
I did some fancy rebasing to split up the interception refactor and debug provider changes, apparently less successfully than I realized.